core/frontend/src/components/kraken/ExtensionModal.vue

```diff
@@ -246,13 +249,17 @@ export default Vue.extend({
   },
   watch: {
     extension() {
-      const [first] = Object.keys(this.extension.versions) ?? ''
-      this.selected_version = first
+      this.selected_version = this.getLatestTag()
     },
   },
   mounted() {
-    const [first] = Object.keys(this.extension.versions) ?? ''
-    this.selected_version = first
+    this.selected_version = this.getLatestTag()
+  },
+  methods: {
+    getLatestTag(): string {
+      const tags = Object.keys(this.extension.versions) ?? []
+      return tags.sort((a, b) => -1 * compare(a, b))[0] ?? ''
```

Comment:
Isn't this the same as ?
```suggestion
      const Object.keys(this.extension.versions)
          .sort((a, b) => compare(b, a))[0] ?? []
```